core/src/draw_target/mod.rs

```diff
@@ -316,8 +319,11 @@ pub trait DrawTarget: Dimensions {
     /// The provided iterator is not required to provide `width * height` pixels to completely fill
     /// the area. In this case, `fill_contiguous` should return without error.
     ///
-    /// This method should not attempt to draw any pixels that fall outside the drawable area of the
-    /// target display. The `area` argument can be clipped to the drawable area using the
+    /// The provided iterator is not required to be limited to `width * height` pixels.
+    /// In this case, `fill_contiguous` should ignore the additional pixels and return without error.
```

Comment:
Thanks for clarifying this! I think this could be merged with the previous paragraph to reduce some repetition.